main.py

```diff
@@ -171,7 +178,7 @@ def create_and_run_bot(parameters, llm_api_key):
         
         job_application_profile_object = JobApplicationProfile(plain_text_resume)
         
-        browser = init_browser()
+        browser = init_browser(parameters['browser'])
```

Comment:
make use of factory pattern

Reply:
Thanks, replaced it with `browser = init_browser(BrowserFactory.get_browser_type())`